doc/contributing.rst

```diff
@@ -175,6 +175,11 @@ MNE-Python specific coding guidelines
 * Avoid unnecessary cosmetic changes if they are not the goal of the PR, this will help keep the diff clean and facilitate reviewing.
 * Use underscores to separate words in non class names: n_samples rather than nsamples.
 * Use CamelCase for class names.
+* Import packages in the order:
+  1. builtin modules
+  1. standard scientific modules (numpy as np, scipy modules, etc)
+  1. other modules
+  1. mne
```

Comment:
ah sorry, I just always use 1s since they get auto-formatted and I don't like having to manually re-number if I insert in the middle. I'll do 1234